pkg/controller/build/osbuildcontroller.go

```diff
@@ -144,12 +160,18 @@ func newOSBuildController(
 		DeleteFunc: ctrl.deleteJob,
 	})
 
+	ctrl.pipelineRunInformer.Informer().AddEventHandler(cache.ResourceEventHandlerFuncs{
```

Comment:
thought (non-blocking): I eventually want to have a completely separate reconciliation loop for MachineOSConfigs, MachineOSBuilds, Jobs, and Pipelines. To be clear, I'm not asking you to do anything here; just mentioning it for posterity.